breba_app/generator_agent/agent.py

```diff
@@ -136,8 +136,8 @@ async def stream(self, spec: str, user_name: str, session_id: str) -> AsyncItera
                         "require_user_input": False,
                         "content": chunk.content,
                     }
-            if mode == "values":
-                logger.info(data["messages"][-1].pretty_repr())
+            # if mode == "values":
```

Comment:
Yes, we can just delete these. This was useful before I had better logging. Now it seems to be in the way most of the time.